src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -154,6 +154,22 @@ impl ScalarFunc {
         }
         Ok(Some(pow))
     }
+
+    #[inline]
+    pub fn truncate_int(&self, ctx: &mut EvalContext, row: &[Datum]) -> Result<Option<i64>> {
+        let x = try_opt!(self.children[0].eval_int(ctx, row));
```

Comment:
Does `u64` needs to be considered?

Reply:
I checked the tidb version,  the truncateInt function only process int64 (https://github.com/pingcap/tidb/blob/f270b105c90973761208bf9a2e79a2ac8ed7ec71/expression/builtin_math.go#L1818) . seems the truncate function will dispatch the u64 to truncateReal ?